daemon/ZilliqaDaemon_AWS.cpp

```diff
@@ -25,16 +25,24 @@
 #include <unistd.h>
 #include <algorithm>
 #include <array>
+#include <boost/program_options.hpp>
+#include <chrono>
 #include <cstdio>
 #include <fstream>
 #include <iostream>
 #include <memory>
 #include <stdexcept>
 #include <string>
+#include <thread>
 #include <unordered_map>
 #include <vector>
 
 using namespace std;
+namespace po = boost::program_options;
+
+#define SUCCESS 0
+#define ERROR_IN_COMMAND_LINE -1
+#define ERROR_UNHANDLED_EXCEPTION -2
```

Comment:
Seems `ERROR_UNHANDLED_EXCEPTION` never be used. Is it exception need to be handled?

Reply:
Prefer inline constexpr functions to function macros. (from coding guideline)

Reply:
Yeah. Fixed.

Reply:
still see macros.